cou/utils/openstack.py

```diff
@@ -287,7 +264,28 @@ def __le__(self, other):
 
     def __repr__(self):
         """Return the representation of CompareOpenStack."""
-        return f"{self.__class__.__name__}<{self.openstack_codenames[self.index]}>"
+        return f"{self.__class__.__name__}<{self.os_release}>"
+
+    @property
+    def os_release(self) -> str:
+        """Return the next OpenStack release codename.
+
+        :return: OpenStack release codename.
+        :rtype: str
+        """
+        return self._os_release
+
+    @os_release.setter
+    def os_release(self, value: str):
+        """Setter of OpenStack release codename.
+
+        :param value: OpenStack release codename.
+        :type value: str
+        :raises ValueError: Raise ValueError if codename is unknown.
+        """
+        if value not in self.openstack_codenames:
+            raise ValueError(f"OpenStack '{value}' is not in '{self.openstack_codenames}'")
+        self._os_release = value
```

Comment:
You need to update index here, since if I do `OpenStackRelease.os_release = "yoga"` is should also update to new index.

Reply:
Good catch. Thanks